model-optimizer/extensions/ops/mvn.py

```diff
@@ -70,8 +70,8 @@ def infer(node: None):
         copy_shape_infer(node)
 
 
-class MVNCaffe(Op):
-    op = 'MVNCaffe'
+class AttributedMVN(Op):
```

Comment:
I recommend to have separate class for `MVNCaffe` and `MeanVarianceNormalization` since they have different attributes and semantic and require different transformations to convert them to MVN.]

Also, rename the transformation classes and files accordingly.

Reply:
MeanVarianceNormalization looks to be misleading, since it is just a longer version of MVN, so I decided to name it MVNOnnx since it is needed only for ONNX.